assets/html/single.php

```diff
@@ -13,6 +13,16 @@
 		<?php include 'templates/head-common.php'; ?>
 		<title>AMP WordPress Theme Static Single Template</title>
 		<link rel="canonical" href="single.html">
+		<style amp-custom>
+			<?php include 'templates/site-css.php'; ?>
+
+			.amp-wp-inline-someimagecoverhash {
```

Comment:
@delawski can you describe what these classes are intending here and why not including in the main CSS file?

Reply:
@kopepasah So this is a tricky part. Some Gutenberg blocks use inline styles, e.g. the `wp-block-cover-image` sets up a custom background image on the container. Also, when you set the text alignment in the editor, it gets added as an inline `style=""` property.

In AMP we're not allowed to use such inline styles, that's why they get 'exported' to the main stylesheet in the page `head`. Actually the way it is happening right now is nothing I've invented, it was something I noticed when turning on the AMP plugin and using Gutenberg. The markup was then supplemented classnames suffixed with unique identifiers which were styled in the `<style amp-custom>. .. </style>`.

Here's an example on such element with a unique classname:

```
<p class="amp-wp-inline-256659ade2cf5854eb8a79d0057ffb1c">… like this one, which is right aligned.</p>
```

Reply:
Ahh, that makes sense. This will definitely be plugin territory for putting the necessary inline styles into the proper stylesheet location.

Reply:
Any inline `style` attributes and (now) `style` elements will all be moved to the `head` by the AMP plugin automatically.